include/galsim/hsm/PSFCorr.h

```diff
@@ -137,6 +137,11 @@ namespace hsm {
      *                           exception.
      * @param ksb_moments_max    Use moments up to ksb_moments_max order for KSB method of PSF
      *                           correction.
+     * @param ksb_sig_weight     The width of the weight function for the KSB method.  Normally,
```

Comment:
Should say whether units = pixels or units = whatever was used for the image ```scale``` parameter.